packages/dev/codemods/src/s1-to-s2/src/codemods/codemod.ts

```diff
@@ -32,7 +33,13 @@ interface Options {
 }
 
 export default function transformer(file: FileInfo, api: API, options: Options) {
-  let j = api.jscodeshift;
+  let j = api.jscodeshift.withParser({
+    parse(source: string) {
+      return recastParse(source, {
+        parser: require('recast/parsers/babel')
```

Comment:
does that mean we need a dependency on `@babel/parser` as well? looks like it is used [here](https://github.com/benjamn/recast/blob/master/parsers/babel.ts#L1) but not declared in their package.json so I guess it's a peer dependency that we need to provide?